examples/fizzbuzz/stack.yaml

```diff
@@ -0,0 +1,3 @@
+resolver: lts-14.12
+extra-deps:
+  - 'hspec-golden-0.1.0.1'
```

Comment:
@Jagl257 Can we include the current status of the package instead of the fixed version? Like we do it here: 
https://github.com/stackbuilders/hspec-golden/blob/master/examples/html/stack.yaml#L2-L3

Reply:
Done.